e2e/utils.go

```diff
@@ -150,6 +141,72 @@ func GetElasticArtifactURL(artifact string, version string, OS string, arch stri
 	return downloadURL, nil
 }
 
+// GetObjectURLFromBucket extracts the media URL for the desired artifact from the
+// Google Cloud Storage bucket used by the CI to push snapshots
+func GetObjectURLFromBucket(bucket string, object string) (string, error) {
+	exp := GetExponentialBackOff(1 * time.Minute)
```

Comment:
It seems surprising to multiply by 1. :) What's the reason for this?

Reply:
It's because the method needs a time.Time argument, and we want to ensure it belongs to the order of minutes. If this is not clear, we can make it more explicit to 60 seconds. Any suggestion is welcome!

Reply:
I added 7038847e56430337aebc7299b24dab36828a77d5, removing the need of multiplying by the neutral element: simply using the default duration. Thanks for this improvement!